omod/src/main/webapp/resources/htmlforms/HMIS-002-OPD.xml

```diff
@@ -0,0 +1,417 @@
+<htmlform formName="HMIS OPD 002 - Outpatient Register" formDescription="Outpatient Register" formEncounterType="ee4780f5-b5eb-423b-932f-00b5879df5ab" formUuid="8c2f8378-51c7-4407-85a0-cb2832accfcf" formVersion="4.0.0" formAddMetadata="yes" formUILocation="patientDashboard.visitActions" formOrder="1" formIcon="icon-medkit" formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.dead==false &amp;&amp; sessionLocation.uuid=='86863db4-6101-4ecf-9a86-5e716d6504e4'" formDisplayStyle="Standard" formLabel="Outpatient(OPD) Form">
```

Comment:
In this Form header, look at the session location id, it must be the uuid of OPD clinic and not ART, In your database look for the Locations table and get the uuid of OPD Clinic

Reply:
@solemabrothers This has been resolved.